bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -189,6 +211,115 @@ const VideoListItem = (props) => {
       </Styled.BottomBar>
     </>
   );
+  const resetEvent = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+  }
+
+  useEffect(() => {
```

Comment:
This whole drag-and-drop thing - IMO it would fit better as a higher-order component of sorts for two reasons:
  - Re-using it in video-preview where it would also be equally useful
  - Preventing video-list-item bloat

Reply:
Probably fixed in https://github.com/bigbluebutton/bigbluebutton/pull/14794/commits/b3d18a1d28b917179bed233c48b0804a1d83e307.